cloudplatform/cloudplatform-connectivity-scp-cf/src/main/java/com/sap/cloud/sdk/cloudplatform/connectivity/ScpCfDestinationRetrievalStrategy.java

```diff
@@ -58,8 +58,15 @@ public String toString()
         return identifier;
     }
 
+    /**
+     * Get the strategy by its identifier.
+     *
+     * @param identifier
+     *            The string representation of a strategy.
+     * @return The strategy enum value.
```

Comment:
Missing `@since` in JavaDoc.

```suggestion
     * @return The strategy enum value.
     * @since 5.0.0
```